### DISCUSSION TITLE
How to use pre-chunked documents as input to graphrag?

### DISCUSSION THREAD

**Discussion by zhangyuanwang777** [microsoft/graphrag#1174] [Q&A] [answered]
I have a pre-chunked document with only 100 tokens per text chunk, and I want to use grapgrag to analyze it. I found that there is an option in the [source code](https://github.com/microsoft/graphrag/blob/main/graphrag/index/verbs/entities/extraction/strategies/graph_intelligence/run_graph_intelligence.py) that defaults to False for prechunked. How can I change it to True in the configuration file? And what data format should I use to input my document data?

**Comment by zhangyuanwang777** [comment, marked as answer]
This problem can be solved by passing data in csv format

**Reply by lyl2000zy** [reply]
What about the code change?Can you give me a specific process,please ?

**Reply by adhazel** [reply]
I found this to be most helpful: https://github.com/microsoft/graphrag/blob/main/docs/config/env_vars.md#input-data

Essentially, prechunked is set by the system and not something you need to set in your vars manually. Instead, just use the CSV format for input and that in of itself will end up setting your prechunked to True. 

Hope this helps.
